.travis.yml

```diff
@@ -8,8 +8,10 @@ dist: trusty
 language: generic
 compiler:
   - gcc
-#notifications:
-#  Whoever receives success and/or failure via email
+notifications:
+  email:
+    recipients:
+      - gm130s@gmail.com
```

Comment:
Sorry, I don't understand your comment. I reverted back the comment line in my updated commit. Is that what you meant?